pubspec.yaml

```diff
@@ -23,12 +23,15 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  provider: ^4.0.0
+  provider: ^6.0.0
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
-  graphql_flutter: 5.0.0
+  graphql_flutter: ^5.0.0
```

Comment:
it will be nice if u use the fix version number. Imagine if latest version has the breaking changes, ur project will not get affected.